.eslintrc.cjs

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true
+  },
+  extends: ['eslint:recommended'],
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module'
+  },
+  ignorePatterns: ['*.generated.js'],
+  rules: {},
+  overrides: [
+    {
+      files: ['*.js', '*.cjs', '*.mjs'],
+      extends: [
+        'standard'
+      ]
+    },
+    {
+      files: ['*.test.js'],
+      plugins: ['mocha'],
+      extends: [
+        'standard',
+        'plugin:mocha/recommended'
+      ],
+      rules: {
+        'mocha/no-skipped-tests': 'error',
+        'mocha/no-setup-in-describe': 'off',
```

Comment:
The default in `plugin:mocha/recommended` is that [`mocha/no-skipped-tests`](https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-skipped-tests.md) and [`mocha/no-exclusive-tests`](https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-exclusive-tests.md) are both set to warnings. So without this change to `.eslintrc`, we would get warnings not errors.

In my experience, when testing a feature, it's pretty common to run tests with `.only` or `.skip`. But you definitely don't want to commit a `.only`-used-for-dev-testing to the repo. So with the error the linter will catch it rather than relying on a reviewer to catch it.

The downside is we need to suppress the legitimate skip errors.

**Note:** IMO, erroring for `.skip` and `.only` is good, but erroring for `.only` is more important. And... the original setup from when you reviewed did not error on `.only`. I've just added that, via `'mocha/no-exclusive-tests': 'error'`.

Reply:
The current approach is fine with me, requiring a bit more effort to make a skipped test, it's fine to discourage that, but in practice we just have a couple of them indeed.